PlaceRouteHierFlow/placer/SeqPair.cpp

```diff
@@ -1214,14 +1225,17 @@ void SeqPair::PerturbationNew(design& caseNL) {
   KeepOrdering(caseNL);
   SameSelected(caseNL);
 
-  // std::string pos("{ "), neg("{ "), sel("{ ");
-  // for (auto& it : posPair) pos += (std::to_string(it) + " ");
-  // for (auto& it : negPair) neg += (std::to_string(it) + " ");
-  // for (auto& it : selected) sel += (std::to_string(it) + " ");
-  // pos += "}";
-  // neg += "}";
-  // sel += "}";
-  // logger->info("sp_before_perturbation {0} {1} {2}", pos, neg, sel);
+  /*if (caseNL._debugofs.is_open()) {
```

Comment:
Probably not, I will try to simplify this. It might still be useful to log the SP before and after a perturbation for debug purposes. There seems to be high amount of unused code in the placer (either commented out or replaced by a newly defined function as also indicated by coverage). A future PR could eliminate all this dead code either commented out or no longer used (never reached). @854768750 , do you want to take a shot at this in the next PR?

Reply:
I can remove some unused code in the next PR.